codegen/testdata/simple/simple.sysl

```diff
@@ -15,6 +15,18 @@ Simple "Simple Server" [package="simple"]:
         GET:
             return ok <: str
 
+    /raw/states:
+        GET:
+            return ok <: str
+
+    /raw/{id<:string}/states [~vars_in_url_name]:
```

Comment:
@anz-gordonj7 Please note here, that the params type should be string for now,  int64 the generated code will get type mismatch error as the GetURLParam from the server-lib just returns a string. Let me know if we want to support int64, it will take more effort to work on the transform conditions.

Reply:
Thanks, but yeah, we need it to support int64 also please

Reply:
No problems, I'll working on this.